ceremonies/src/mock.rs

```diff
@@ -47,11 +48,20 @@ frame_support::construct_runtime!(
 	}
 );
 
+parameter_types! {
+	pub const ReputationLifetime: u32 = 1;
+	pub const AmountNewbieTickets: u8 = 50;
+	pub const MinSolarTripTimeS: i32 = 1;
+	pub const DefaultDemurrage: i128 	 = 0x0000000000000000000001E3F0A8A973_i128;
```

Comment:
fmt: cargo fmt has problems detecting fmt errors in macros because it can't always know if a fmt change, would also change the macros output.